documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation.rst

```diff
@@ -0,0 +1,17 @@
+.. _lesion-segmentation:
+
+Lesion segmentation in spinal cord injury (SCI) or multiple sclerosis (MS)
+##########################################################################
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/sciseg.png
+  :align: center
+  :figwidth: 60%
+
+This section demonstrates how to segment lesions in spinal cord injury (SCI) or multiple sclerosis (MS) using the ``sct_deepseg -task seg_sc_lesion_t2w_sci`` or ``sct_deepseg_lesion`` commands, respectively.
```

Comment:
I think you can also mention `sct_deepseg -task seg_sc_ms_lesion_stir_psir` and `sct_deepseg -task seg_ms_lesion_mp2rage`

Reply:
Good point. Added in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/c6c11c0e857f05b1c78fbf134685a8bee2e09243